backend/capellacollab/settings/modelsources/t4c/routes.py

```diff
@@ -101,5 +101,5 @@ def edit_t4c_instance(
     for key in body.dict():
         value = body.__getattribute__(key)
         if value:
-            instance.__setattr__(key, body.__getattribute__(key))
+            instance.__setattr__(key, value)
```

Comment:
You could even use the walrus operator here if you wanted (introduced in Python 3.8). I find it elegant, but it’s not widespread
```python
if value := body.__getattribute__(key):
    instance.__setattr__(key, value)
```

Reply:
Absolutely! I like it, but always forget to use it :D